### DISCUSSION TITLE
gltf node with simetric transformation aren't display in AR with safari on ios

### DISCUSSION THREAD

**Discussion by denis-bolomier-pch** [google/model-viewer#2885] [Q&A]
### Description

When Im loading a gltf, all node that have been reverse along an axe doesn't display in the AR view on IOS

the model in my Iphone
![IMG_5666](https://user-images.githubusercontent.com/89996185/137627163-0d7d6c6c-86a4-4210-b81c-9270171a6c6c.PNG)

the same model in ar mode
![IMG_5667](https://user-images.githubusercontent.com/89996185/137627205-c5d00eae-3856-4a7c-a23d-24c7afa9284d.JPG)

As you can see the following pyramid aren't display:
-  the front, bottom,right
-  the up, back, right

As you can see the following pyramid are display in the wrong orientation:
- front, top, left

The gltf have been created with a mesh called several time with a different transformation.
[TEST REVERSE.gltf.zip](https://github.com/google/model-viewer/files/7359876/TEST.REVERSE.gltf.zip)

#### Live Demo
https://v.pch.click/viewer.html?short_id=PREQV

### Version
https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js
- model-viewer: v1.8.0

### Browser Affected
- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [X] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
we use the following syntax to have AR
<model-viewer id="model-viewer" src="" ar ar-modes="webxr scene-viewer quick-look" camera-controls alt="A 3D model carousel">

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
IIRC this is a deficiency in the  USDZ format (or at least the way Quick Look interprets it). They don't appear to switch their back-face for a mirroring transform. The only way three.js could work  around this is  to duplicate geometry, which is complicated and inefficient. Please file a bug on Apple. FYI @mrdoob.
